game2048/dbconnection/server.js

```diff
@@ -1,29 +1,56 @@
 const express = require('express');
 const path = require('path');
 const { Client } = require('pg');
+const config = require('./config.json');
+const { saltHashPassword, comparePasswords } = require('./bcrypt-password-hash/passwordHashing');
 
 const app = express();
 const port = 3001;
 
-// PostgreSQL-Verbindung konfigurieren
-const client = new Client({
-    user: 'postgres',
-    host: 'localhost',
-    database: 'userscoresmanagerdb',
-    password: '4fortnite',
-    port: 5432,
-});
+// middleware
+app.use(express.json());
+app.use(express.static(path.join(__dirname, 'public')));
```

Comment:
can you add in readme that you need to create a config and what needs to be in config to work